### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by ragangithub**
- Created a database named vet clinic.
- Created a table named Animals with the different columns
- Inserted data into the table
- Queried data from table

Screenshot (270)
Screenshot (269)

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Ragan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented; Ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe.

**Kindly remove/update all aspects similar to the underlined:**

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ragangithub** [comment]
- removed all  guide instructions

**Review by rloterh — COMMENTED** [review]
Hi Ragan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented; Ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe.

**Kindly remove/update all aspects similar to the underlined:**

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ragangithub** [comment]
- removed  guide instructions

File: README.md [hunk 1]
```diff
@@ -1 +1,269 @@
...
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
```
Comment (Munsa1) [inline README.md]: - [ ] Kindly remove commented lines from your readme file as per requirement in line 32. This goes for lines commented through out the readme file which have no relevance

File: data.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', 0, 'true', 10.23);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, 'true', 8.00);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', 1, 'false', 15.04);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, 'false', 11);
```
Comment (Munsa1) [inline data.sql]: - [ ] For `Devimon` the value for `neutered` should be set to `true` as per requirement.
image
check requirements [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md)

## Important 🔴 
### Note that the screenshots have to be udated for all queries where changes are required

File: queries.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+
+SELECT * FROM animals WHERE name like '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (Munsa1) [inline queries.sql]: - [ ] Here we are supposed to select the `name` of all animals born between 2016 and 2019 and not the entire record.
e.g
`SELECT name FROM animals`

NOT ❌ 
`SELECT * FROM animals`
- [ ] Also the date should be between `'2016-01-01' AND '2019-12-31'` and not `'2016-01-01' AND '2019-01-01'`

File: queries.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+SELECT * FROM animals WHERE neutered = 'true' AND  escape_attempts < 3;
```
Comment (Munsa1) [inline queries.sql]: - [ ] Again here we are supposed to select the `name` column only not the entire record
image
Check the requirements

File: README.md [hunk 5]
```diff
@@ -1 +1,269 @@
...
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
```
Comment (Munsa1) [inline README.md]: - [ ] Kindly remove this link from the table of contents as it points to a section which has been removed

File: README.md [hunk 6]
```diff
@@ -1 +1,269 @@
...
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details> -->
+
+<!-- <details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details> -->
```
Comment (Munsa1) [inline README.md]: - [ ] Again here kindly remove commented lines for a cleaner more professional readme

File: README.md [hunk 7]
```diff
@@ -1 +1,269 @@
...
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](<replace-with-your-deployment-URL>)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
```
Comment (Munsa1) [inline README.md]: - [ ] Again here kindly remove commented lines for a cleaner more professional readme

File: README.md [hunk 8]
```diff
@@ -1 +1,269 @@
...
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
```
Comment (Munsa1) [inline README.md]: - [ ] Again here kindly remove commented lines for a cleaner more professional readme

File: README.md [hunk 9]
```diff
@@ -1 +1,269 @@
...
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
```
Comment (Munsa1) [inline README.md]: - [ ] Again here kindly remove commented lines for a cleaner more professional readme

File: README.md [hunk 10]
```diff
@@ -1 +1,269 @@
...
+👤 **Ragan Hailu**
+
+- GitHub: [Ragan](https://github.com/ragangithub)
+- LinkedIn: [Ragan Hailu](https://linkedin.com/in/raganhailu)
+
+<!-- 👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle) -->
```
Comment (Munsa1) [inline README.md]: - [ ] Again here kindly remove commented lines for a cleaner more professional readme

File: README.md [hunk 11]
```diff
@@ -1 +1,269 @@
...
+
+<!-- SUPPORT -->
+
+<!-- ## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
```
Comment (Munsa1) [inline README.md]: - [ ] Again here kindly remove commented lines for a cleaner more professional readme

File: README.md [hunk 12]
```diff
@@ -1 +1,269 @@
...
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
```
Comment (Munsa1) [inline README.md]: - [ ] Again here kindly remove commented lines for a cleaner more professional readme

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @ragangithub, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Clean Codebase :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Followed Given File Structure :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ragangithub** [comment]
- Removed comments from README
- Changed   Devimon's neutered value to true
- Changed date value from 2019-12-31' to'2019-01-01'
- Displayed the names only as suggested by the reviewer
Screenshot (270)

Screenshot (269)

File: README.md (hunk 5 above)
Comment (danmainah) [reply README.md]: - [ ] @ragangithub Great job implementing the changes requested, however you forgot to remove the links to the FAQ section that is not there in your readme file

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ragangithub , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️  Implemented changes requested to ensure the use of the right data and correct queries.
✔️ Professional written README file with all sections required.


 # Required Changes ♻️

- [ ] @ragangithub Great work showing the updated screenshots of the changes requested however I would recommend you update the screenshots queries in the pull request summary as previously requested.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ragangithub** [comment]
- removed FAQ section
- updated screenshot for queries
Screenshot (270)
Screenshot (269)

**Review by OyePriscilla — APPROVED** [review]
Hi @ragangithub 👍, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Added query's result to the PR ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**